src/components/item/form/DocumentForm.tsx

```diff
@@ -77,24 +106,48 @@ export const DocumentExtraForm = ({
           </Select>
         </FormControl>
       </Box>
-      <Box sx={{ mt: 2 }}>
-        <DocumentItem
-          edit
-          id={documentItemId}
-          item={{
-            extra: buildDocumentExtra(extra),
-          }}
-          maxHeight={maxHeight}
-          onCancel={onCancel}
-          onChange={onContentChange}
-          onSave={onContentSave}
-          placeholderText={placeholder}
-          saveButtonId={saveButtonId}
-          cancelButtonId={cancelButtonId}
-          showActions={showActions}
-        />
-      </Box>
-    </>
+      <TabContext value={editorMode.toString()}>
+        <Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
+          <TabList
+            onChange={(_, value) => handleChangeEditorMode(value)}
+            aria-label="lab API tabs example"
+          >
+            <Tab
+              label={t(BUILDER.DOCUMENT_EDITOR_MODE_RICH_TEXT)}
+              value={EditorMode.Rich.toString()}
+            />
+            <Tab
+              label={t(BUILDER.DOCUMENT_EDITOR_MODE_RAW)}
+              value={EditorMode.Raw.toString()}
```

Comment:
Is `toString()` always necessary?

Reply:
Since the tab switch only handles strings yes.